lib/auth/accountrecovery_test.go

```diff
@@ -1334,3 +1378,25 @@ func getMockedU2FAndRegisterRes(authSrv *Server, tokenID string) (*proto.U2FRegi
 		ClientData:       u2fRegResp.ClientData,
 	}, u2fKey, nil
 }
+
+func getMockedU2FAndRegisterRes(authSrv *Server, tokenID string) (*wantypes.CredentialCreationResponse, *mocku2f.Key, error) {
+	res, err := authSrv.CreateRegisterChallenge(context.TODO(), &proto.CreateRegisterChallengeRequest{
```

Comment:
Use context.Background unless you have intention to change it (there is nothing TODO otherwise). Since this is a new method, it should either already take a ctx or default to context.Background.

```suggestion
	res, err := authSrv.CreateRegisterChallenge(ctx, &proto.CreateRegisterChallengeRequest{
```